### DISCUSSION TITLE
how to skip domain validation if using nextcloud aio through virtual box vm?

### DISCUSSION THREAD

**Discussion by sanfx** [nextcloud/all-in-one#3526] [Questions] [answered]
after I loaded the Nextcloud-AIO.ova I downloaded from https://download.nextcloud.com/aio-vm/

I started the vm. I can access the IP:8080/login it gives me password to login then I end up at page IP:8080/containers

I wanted to skip the domain validation as I know have domain setup working using duckdns.org 

https://github.com/nextcloud/all-in-one#how-to-skip-the-domain-validation

the above link says: to `--env SKIP_DOMAIN_VALIDATION=true`

but the container is already running and I do not know where the start.sh script is that starts nextcloud on vm boot up.

so how should I add --env SKIP_DOMAIN_VALIDATION=true to the container?

**Comment by szaimen** [comment, marked as answer]
As written, stop the mastercontainer, remove it and recreate it with the adjusted command.

btw, did you follow the readme in the download folder?

**Reply by sanfx** [reply]
i am novice in docker, I do not know how can I adjust docker command , would have been nice if it was in a file or docker compose.
